Remove call to poison_init_mem() in arch/arm/mm/init.c because it breaks Clang

---
 init.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Index: linux/arch/arm/mm/init.c
===================================================================
--- linux.orig/arch/arm/mm/init.c	2012-09-03 11:29:39.946606904 -0400
+++ linux/arch/arm/mm/init.c	2012-09-05 12:13:49.229569888 -0400
@@ -743,7 +743,8 @@
 				    "TCM link");
 #endif
 
-	poison_init_mem(__init_begin, __init_end - __init_begin);
+	// FIXME - This causes a data abort, fails part way through
+	//poison_init_mem(__init_begin, __init_end - __init_begin);
 	if (!machine_is_integrator() && !machine_is_cintegrator())
 		totalram_pages += free_area(__phys_to_pfn(__pa(__init_begin)),
 					    __phys_to_pfn(__pa(__init_end)),
